src/extensions/default/bramble/stylesheets/sidebarTheme.css

```diff
@@ -21,6 +21,12 @@ div.working-set-option-btn[style] {
     text-shadow: none;
 }
 
+/* File bar hover styles */
+.sidebar-hover {
+  background-color: #333533 !important;
```

Comment:
make sure to follow the indentation rules that you see used in the rest of the file: this file has four-space indentation